### ISSUE TITLE
Substantial slowness in Immich project

### ISSUE THREAD

**Issue by benmccann** [sveltejs/eslint-plugin-svelte#1552]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

10.2.1

### What version of `eslint-plugin-svelte` are you using?

3.19.0

### What did you do?

```sh
git clone git@github.com:immich-app/immich.git
cd immich/web
pnpm install
pnpm run lint
```

### What did you expect to happen?

Finish in under a couple of minutes

### What actually happened?

Takes several minutes to half an hour depending on how powerful of a machine you run it on

### Link to **GitHub Repo** with Minimal Reproducible Example

https://github.com/immich-app/immich (see the `web` directory)
